src/mapml/control/AttributionButton.js

```diff
@@ -0,0 +1,110 @@
+export var AttributionButton = L.Control.extend({
```

Comment:
It looks really clean imo.

One suggestion is if it extends Leaflet's Attribution, you can extend Attribution directly and simply overwrite the functions you need changed.

```js
export var AttributionButton = L.Attribution.extend({...
```

Reply:
I wasn't able to get that to work, feel free to make the suggested changes.